graphics/graphics_frames.xml

```diff
@@ -754,15 +758,15 @@
 
 	<!-- Tokens -->
 	<framedef name="token_name">
-		<bitmap file="graphics/pngs/mini_name.png" />
+		<bitmap file="graphics/pngs/mini_name_height.png" />
 		<offset>7,7,7,7</offset>
 	</framedef>
 	<framedef name="token_ordinal">
 		<bitmap file="graphics/pngs/tokennumber.png" />
 		<offset>6,6,6,6</offset>
 	</framedef>
 	<framedef name="token_height">
-		<bitmap file="graphics/pngs/mini_name.png" />
+		<bitmap file="graphics/pngs/mini_name_height.png" />
```

Comment:
These two are temporary fixes for the token height and name problems with the latest version of FGU.